package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -70,6 +70,15 @@ not reflect in the results.
 
 .. versionadded:: 0.19.0
 
+Important Note
```

Comment:
I have only left a note, would a warning be useful too? (not really sure how you do warnings in a cython file).

Reply:
Do you mean `import warnings`? I _think_ that should work.

Can you also add a warning in the markup
```rest
.. warning::

   The current nsgrid code can return incorrect values. Do not use for production use.
   See issue `#2930<https://github.com/MDAnalysis/mdanalysis/issues/2930>`_ for further context.
```
(I hope that's the right [syntax for inline reST links](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links)).